src/transformers/models/idefics2/modeling_idefics2.py

```diff
@@ -1805,15 +1801,16 @@ def forward(
         >>> prompts = [
         ...   "<image>In this image, we can see the city of New York, and more specifically the Statue of Liberty.<image>In this image,",
         ...   "In which city is that bridge located?<image>",
-        >>> ]
+        ... ]
         >>> images = [[image1, image2], [image3]]
-        >>> inputs = processor(text=prompts, padding=True, return_tensors="pt").to(DEVICE)
+        >>> inputs = processor(text=prompts, padding=True, return_tensors="pt").to("cuda")
 
         >>> # Generate
-        >>> generated_ids = model.generate(**inputs, bad_words_ids=BAD_WORDS_IDS, max_new_tokens=500)
+        >>> generated_ids = model.generate(**inputs, bad_words_ids=BAD_WORDS_IDS, max_new_tokens=20)
         >>> generated_texts = processor.batch_decode(generated_ids, skip_special_tokens=True)
 
         >>> print(generated_texts)
+        ['In this image, we can see the city of New York, and more specifically the Statue of Liberty. In this image, we can see the city of New York, and more specifically the Statue of Liberty.\n\n', 'In which city is that bridge located?\n\nThe bridge is located in the city of Pittsburgh, Pennsylvania.\n\n\nThe bridge is']
```

Comment:
The results are not very good. But we don't have it previously, so I don't know if we consider it bad.

Reply:
I'm surprised by these generations - they also don't look like the outputs when I was integrating the model 🤔

Reply:
maybe you were using other type of GPUs. CI is using T4.

Reply:
Quite possibly - I was using `NVIDIA A10G`. I find it quite concerning the generations can be so much worse between hardware though, especially given this isn't a tiny model

Reply:
I can check it tomorrow. But I will merge first 🙏